package.json

```diff
@@ -8,7 +8,7 @@
     "test": "test"
   },
   "scripts": {
-    "build": "node script/build.js && npm run compile-sass && npm run autoprefix",
+    "build": "rm -rf lib/{js,css} && npx babel src/js --out-dir lib/js && npm run compile-sass && npm run autoprefix",
```

Comment:
I'm assuming that all the js and css files are expendable in `lib/`. If we don't remove them forcibly then old copies of removed files would remain around unless we removed them explicitly.

Reply:
Looks like this was a feature request at some point, but seems like it didn't go anywhere... https://github.com/babel/babel/issues/5636